content/departments/engineering/teams/core-services/managed-services/telemetry-gateway.md

```diff
@@ -0,0 +1,80 @@
+# Telemetry Gateway
+
+The Telemetry Gateway service is the service that ingests [telemetry v2 events](https://docs.sourcegraph.com/dev/background-information/telemetry) from all Sourcegraph instances, and is available at `telemetry-gateway.sourcegraph.com`.
+
+- As of 5.2.0, [certain flags can be configured](https://docs.sourcegraph.com/dev/background-information/telemetry#enabling-telemetry-export) to export events that have been instrumented with the new APIs to Telemetry Gateway.
+- For Sourcegraph instances that prior to 5.2.0, no events are exported.
```

Comment:
Should we mention Cloud instances have been exporting event logs through other means since version X based on individual service agreement?

Reply:
Added!